README.md

```diff
@@ -80,77 +80,82 @@ The platform is built on top of [Ambire Wallet](https://www.ambire.com) and uses
 
 ### Values
 
-* **Transparency:** full reporting transparency for publishers and advertisers; they both receive the same reports
-* **No intermediaries:** connect publishers and advertisers as directly as possible, therefore maximizing results and revenues; this also implies minimized fees and no commissions
-* **Privacy:** ensure the user's data stays private by never collecting it
-* **No custody of funds:** users have full control over their own funds; you can withdraw any amount at any point, no thresholds
-* **Censorship resistance:** we empower a free and self-governed ad market with no centralised restrictions on what can and can't be advertised
-* **Ease of use:** modern adtech is complicated. We aim to make it as easy as possible to interact with AdEx
-* **Flexibility:** wide variety of use cases, including but not limited to: display advertising, affiliate networks, and even content micropayments
+* **Transparency** Transparency is ensured through a decentralized ledger that records ad transactions, enabling a verifiable trail of information and minimizing fraud.
+* **Low fees** We only ask for a 7% network fee, which is necessary to operate the platform on the blockchain. Compared to other DSPs with 15-20% fees, our platform significantly reduces costs by over 50%. In addition, the fee for advertisers, who use the ADX token to launch a campaign is only 4%. 
```

Comment:
this is misleading, it sounds like it's a blockchain fee rather than the fee of the AdEx network itself. Even though I came up with "network fee" for this, I think now that it's a misleading term and should be revised.